extensions/resteasy-reactive/jaxrs-client-reactive/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/JaxrsClientReactiveProcessor.java

```diff
@@ -325,6 +348,180 @@ public void accept(EndpointIndexer.ResourceMethodCallbackData entry) {
                     .produce(new ReflectiveClassBuildItem(true, false, false, writerClass));
         }
 
+        Map<String, RuntimeValue<MultipartResponseData>> responsesData = new HashMap<>();
+        for (ClassInfo multipartResponseType : multipartResponseTypes) {
+            responsesData.put(multipartResponseType.toString(), createMultipartResponseData(multipartResponseType,
+                    generatedClassBuildItemBuildProducer, recorderContext));
+        }
+        recorder.setMultipartResponsesData(responsesData);
+    }
+
+    private RuntimeValue<MultipartResponseData> createMultipartResponseData(ClassInfo multipartResponseTypeInfo,
```

Comment:
Can you provide a comment on the method about what it does?

Reply:
done, I hope it helps

Reply:
Great, thanks!